From a92cbc0715bc242e2f61091d3424f127d85bd8f6 Mon Sep 17 00:00:00 2001
From: Raphael Geissert <geissert@debian.org>
Date: Fri, 16 Sep 2016 16:01:03 +0200
Subject: [PATCH 2/2] apt.systemd.daily: set the timezone to UTC

Several DST-related problems affect the check_stamp checks. By switching
to UTC, they go away.

Closes: #504106
---
 debian/apt.systemd.daily |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/debian/apt.systemd.daily b/debian/apt.systemd.daily
index a952954..3da9fab 100755
--- a/debian/apt.systemd.daily
+++ b/debian/apt.systemd.daily
@@ -343,6 +343,9 @@ if which apt-get >/dev/null 2>&1 && ! eval apt-get check $XAPTOPT $XSTDERR ; the
     exit 0
 fi
 
+# Set the timezone to UTC to avoid DST-related problems (#504106)
+TZ=UTC
+export TZ
 # Global current time in seconds since 1970-01-01 00:00:00 UTC
 now=$(date +%s)
 
-- 
1.7.2.5

